src/calc-length.js

```diff
@@ -79,6 +79,20 @@
     return new CalcLength(calcDictionary);
   };
 
+  CalcLength.prototype.divide = function(divider) {
+    var calcDictionary = {};
+
+     //iterate through all length types and divide all non null lengths 
```

Comment:
Nitpick: Comment should start with capital, should have one space between the // and the text